pydantic/_internal/_generate_schema.py

```diff
@@ -1444,7 +1444,6 @@ def _sequence_schema(self, sequence_type: Any) -> core_schema.CoreSchema:
         item_type = self._get_first_arg_or_any(sequence_type)
         item_type_schema = self.generate_schema(item_type)
         list_schema = core_schema.list_schema(item_type_schema)
-        metadata = build_metadata_dict(initial_metadata={'known_metadata_as': typing.Sequence})
```

Comment:
This was driving me nuts - I think we're trying to stray away from excessive use of metadata flags, so I've removed this logic.